src/verifier.rs

```diff
@@ -69,6 +69,13 @@ impl SDJWTVerifier {
         verifier.verified_claims = verifier.extract_sd_claims()?;
 
         if let (Some(expected_aud), Some(expected_nonce)) = (&expected_aud, &expected_nonce) {
+            let sign_alg = verifier.sd_jwt_engine.unverified_input_key_binding_jwt
+                .clone()
```

Comment:
cloned value here is used by ref below. Can it be siplified?

Reply:
Removed cloning & use reference instead